code/modules/mob/living/damage_procs.dm

```diff
@@ -7,15 +7,13 @@
 	* penetration: How much the damage source might bypass the armour value (optional)
 	* def_zone: What part of the body we want to check the armor of (optional)
 
-	Hard armor is calculated as a flat reduction before soft armor is applied as a multiplier.
-	Penetration reduces soft armor by a flat amount, but reduces hard armor as a multiplier.
+	Hard armor reduces penetration or sunder a flat amount, and sunder in the case of xenos
```

Comment:
Reduces penetration or sunder, and sunder.

Reply:
done, and done